src/View/TwigView.php

```diff
@@ -19,14 +19,36 @@
 namespace Cake\TwigView\View;
 
 use Cake\Core\Configure;
-use Cake\TwigView\Event\ConstructEvent;
-use Cake\TwigView\Event\EnvironmentConfigEvent;
-use Cake\TwigView\Event\LoaderEvent;
+use Cake\Core\Plugin;
+use Cake\TwigView\Twig\Extension\ArraysExtension;
```

Comment:
This use list for extensions is a bit too long. We could instead just use `use Cake\TwigView\Twig\Extension` and then extensions as `new Extension\BasicExtension()`.

Reply:
done.